package.json

```diff
@@ -86,7 +86,7 @@
   ],
   "scripts": {
     "test": "npm run test:jest",
-    "test:all": "npm run test:jest && npm run test:tape && npm run test:native && npm run test:pure && npm run test:typescript && npm run test:fetch",
+    "test:all": "npm run test:jest && npm run test:tape && npm run test:native && npm run test:pure && npm run test:typescript && npm run test:fetch && npm run test:jsdom",
```

Comment:
nit: why not run these in parallel?

Reply:
not sure, guessing so that the test output is not interleaved? the tests are very fast here anyway